partcad-cli/src/partcad_cli/click/commands/search/all.py

```
@@ -0,0 +1,58 @@
+import rich_click as click
+from click.testing import CliRunner
+
+from partcad.context import Context
+import partcad.logging as pc_logging
```

Ruff (0.8.2)

`partcad.logging` imported but unused

Remove unused import: `partcad.logging`

(F401)